<?xml version="1.0"?>
<robot name="mrpiz">

 <link name="base_link">
   <visual>
    <origin rpy="0 0 0" xyz="0 0 0.0175"/>
    <geometry>
     <cylinder length="0.035" radius="0.05"/>
    </geometry>
    <material name="blue">
      <color rgba="0 0 0.8 1"/>
    </material>
   </visual>
 </link>

<link name="right_wheel"> 
    <visual> 
      <origin rpy="0 -1.5708 0" xyz="0.05 0 0.016"/> 
      <geometry> 
        <cylinder length="0.005" radius="0.016"/> 
      </geometry> 
      <material name="black"> 
        <color rgba="0 0 0 1"/> 
      </material> 
    </visual> 
  </link>

<joint name="right_wheel_joint" type="fixed"> 
    <axis xyz="0 0 1"/> 
    <parent link="base_link"/> 
    <child link="right_wheel"/> 
  </joint>

<link name="left_wheel"> 
    <visual> 
      <origin rpy="0 -1.5708 0" xyz="-0.05 0 0.016"/> 
      <geometry> 
        <cylinder length="0.005" radius="0.016"/> 
      </geometry> 
      <material name="black"> 
        <color rgba="0 0 0 1"/> 
      </material> 
    </visual> 
  </link>

  <joint name="left_wheel_joint" type="fixed"> 
    <axis xyz="0 0 1"/> 
    <parent link="base_link"/> 
    <child link="left_wheel"/> 
  </joint>

 <link name="pi_board">
        <visual>
            <origin rpy="0 0 0" xyz="0 0 0.035"/> 
            <geometry>
              <box size="0.03 0.065 0.005" />
            </geometry>
            <material name="green_pcb">
                <color rgba="0.23 0.4 0.16 1.0"/>
            </material>
        </visual>
    </link>

    <joint name="base_to_pi_board" type="fixed">
        <parent link="base_link"/>
        <child link="pi_board"/>

    </joint>


</robot>
